package.json

```diff
@@ -155,11 +155,5 @@
   },
   "config": {
     "wp_org_slug": "woocommerce-services"
-  },
-  "husky": {
```

Comment:
No longer necessary. See https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v7